tests/test_project.py

```diff
@@ -104,7 +104,23 @@ def test_path_to_local_project_can_be_specified(self):
         # local() is called more than once so we need an extra next_call()
         # otherwise fudge compares the args to the last call to local()
         self.fake_local.with_args(
-            arg.endswith("-C %s %s" % os.path.split(project_path))
+            arg.endswith("-C path/to/my project")
```

Comment:
tangential commentary: another unit-test pitfall is duplicating the code being tested in the test. In this case it works, but if it didn't, it would still pass ...